### PR TITLE
Upgrade gradle to 6.2.1

### PR DESCRIPTION
**Pull request by tokuhirom**
- switch to `implementation` from `compile`

### CODE REVIEW

File: sample-spring-boot-kitchensink/build.gradle [hunk 1]
```diff
@@ -17,5 +17,8 @@
 apply plugin: 'org.springframework.boot'
 
 dependencies {
-    compile project(':line-bot-spring-boot')
+    implementation project(':line-bot-spring-boot')
+    implementation "org.springframework.boot:spring-boot-starter-web"
+    implementation 'com.google.guava:guava'
+
```
Comment (imasahiro) [inline sample-spring-boot-kitchensink/build.gradle]: nit. remove extra newline

File: line-bot-spring-boot/build.gradle [hunk 2]
```diff
@@ -14,12 +14,15 @@
...
-    compile project(':line-bot-api-client')
-    compile project(':line-bot-model')
-    compile project(':line-bot-servlet')
-    compile 'org.springframework.boot:spring-boot-autoconfigure'
-    compile 'org.springframework.boot:spring-boot-starter-web'
+    api project(':line-bot-api-client')
+    implementation project(':line-bot-model')
+    implementation project(':line-bot-servlet')
+    implementation 'org.springframework.boot:spring-boot-autoconfigure'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```
Comment (kazuki-ma) [inline line-bot-spring-boot/build.gradle]: `line-bot-spring-boot` uses `org.springframework.boot:spring-boot-starter-web` as it's api, not internal implementation only.
So use `api` here.

File: line-bot-api-client/build.gradle [hunk 3]
```diff
@@ -13,13 +13,16 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+apply plugin: 'java-library'
```
Comment (kazuki-ma) [inline line-bot-api-client/build.gradle]: How about replace https://github.com/line/line-bot-sdk-java/pull/446/files/3a8efdb046303f62ee575d1329be7c70497c971e#diff-c197962302397baf3a4cc36463dce5eaR79 this line?

Comment (kazuki-ma) [reply line-bot-api-client/build.gradle]: link miss. here

https://github.com/line/line-bot-sdk-java/blob/3a8efdb046303f62ee575d1329be7c70497c971e/build.gradle#L79

File: line-bot-spring-boot/build.gradle (hunk 2 above)
Comment (tokuhirom) [reply line-bot-spring-boot/build.gradle]: Yep... But every line-bot-spring-boot users should depend on `spring-boot-starter-web`.
As a result, spring-boot-starter-web should be `implementation` here.

File: line-bot-api-client/build.gradle (hunk 3 above)
Comment (tokuhirom) [reply line-bot-api-client/build.gradle]: ok. I'll do this

File: line-bot-spring-boot/build.gradle (hunk 2 above)
Comment (kazuki-ma) [reply line-bot-spring-boot/build.gradle]: ok

**Pull request merged**